lib/group/group-router.js

```diff
@@ -1,56 +1,75 @@
 'use strict';
 
 var express = require('express'),
-  config = require('./config-group');
+  config = require('./config-group'),
+  shortid = require('shortid');
 
-function initRouter(mediator) {
+var Topics = require('fh-wfm-mediator/lib/topics');
+
+function initRouter(topics) {
   var router = express.Router();
 
   router.route('/').get(function(req, res) {
-    mediator.once('done:wfm:cloud:group:list', function(data) {
-      res.json(data);
-    });
-    mediator.publish('wfm:cloud:group:list');
+    topics.request('list')
+      .then(function(data) {
+        res.json(data);
+      })
+      .catch(function(err) {
+        throw err;
```

Comment:
I will add that unless there's middleware to log all 500 errors, then you probably want to log this err somewhere so you know the client saw an error.